src/main/scala/io/iohk/praos/App.scala

```diff
@@ -85,11 +87,14 @@ object App extends Logger {
       /**
         * @note Applies the new transactions from the received blocks that belongs to the largest chain.
         */
-      (slotState, blockchainState) = ledger.slotEnd(slotState, blockchainState)
+      val (slotState1, blockchainState1) = ledger.slotEnd(slotState, blockchainState)
+      slotState = slotState1
```

Comment:
As this seems to be the only reassigment of the `slotState` variable, maybe it could be a `val` and replace the use of `slotState` with `slotState1` in the next `slotState` uses?

Reply:
:+1: